### PR TITLE
Addressed various Doxygen grammar issues.

### PR DESCRIPTION
**Pull request by brtnfld**
Doxygen Doc. Changes.

### CODE REVIEW

**Review by byrnHDF — APPROVED** [review]

File: src/H5Cpkg.h [hunk 1]
```diff
@@ -4398,7 +4398,7 @@ typedef struct H5C_tag_info_t {
  * order, we scan the index repeatedly, once for each flush dependency
  * height in increasing order.
  *
- * This operation is complicated by the fact that entries other the the
+ * This operation is complicated by the fact that entries other the
```
Comment (lrknox) [inline src/H5Cpkg.h]:
```suggestion
 * This operation is complicated by the fact that entries other than the
```

File: src/H5Lpublic.h [hunk 2]
```diff
@@ -209,7 +209,7 @@ H5_DLL herr_t H5Lmove(hid_t src_loc, const char *src_name, hid_t dst_loc, const
  *
  *          H5Lcopy() retains the creation time and the target of the original
  *          link. However, since the link may be renamed, the character
- *          encoding is that specified in \p lcpl_id rather than that of the
+ *          encoding is  specified in \p lcpl_id rather than that of the
```
Comment (lrknox) [inline src/H5Lpublic.h]:
```suggestion
 *          encoding is specified in \p lcpl_id rather than in that of the
```

File: src/H5MF.c [hunk 3]
```diff
@@ -3236,7 +3236,7 @@ H5MF_settle_meta_data_fsm(H5F_t *f, hbool_t *fsm_settled)
          *           multi file drivers, as the self referential free space
          *           manager header and section info can be stored in up to
          *           two different files -- requiring that up to two EOA's
-         *           be stored in the the free space managers super block
+         *           be stored in the free space managers super block
```
Comment (lrknox) [inline src/H5MF.c]:
```suggestion
         *           be stored in the free space manager's superblock
```

File: src/H5Opublic.h [hunk 4]
```diff
@@ -790,9 +790,9 @@ H5_DLL herr_t H5Olink(hid_t obj_id, hid_t new_loc_id, const char *new_name, hid_
  *          H5Odecr_refcount() should be used.
  *
  *          An object’s reference count is the number of hard links in the
- *          file that point to that object. See the “Programming Model”
+ *          file that points to that object. See the “Programming Model”
```
Comment (lrknox) [inline src/H5Opublic.h]: I think this should be 'point':  it's the hard links that point, not the file.

File: src/H5Spublic.h [hunk 5]
```diff
@@ -293,7 +293,7 @@ H5_DLL hid_t H5Screate(H5S_class_t type);
  *       H5Sselect_elements() selects array elements in a dataspace. These
  *       functions are used for subsetting. H5Sselect_none() removes all
  *       selections from a dataspace and is used in Parallel HDF5 when a process
- *       does not have or need to write data.
+ *       does not have or needs to write data.
```
Comment (lrknox) [inline src/H5Spublic.h]: I don't know the rule for this, but the subject is process and the verbs with number agreement are does have or does need.  I don't think process does not needs is correct.

File: src/H5Tenum.c [hunk 6]
```diff
@@ -333,7 +333,7 @@ H5Tenum_nameof(hid_t type, const void *value, char *name /*out*/, size_t size)
 /*-------------------------------------------------------------------------
  * Function:	H5T__enum_nameof
  *
- * Purpose:	Finds the symbol name that corresponds the the specified
+ * Purpose:	Finds the symbol name that corresponds the specified
```
Comment (lrknox) [inline src/H5Tenum.c]:
```suggestion
 * Purpose:	Finds the symbol name that corresponds to the specified
```

File: src/H5Tenum.c [hunk 7]
```diff
@@ -470,7 +470,7 @@ H5Tenum_valueof(hid_t type, const char *name, void *value /*out*/)
 /*-------------------------------------------------------------------------
  * Function:	H5T__enum_valueof
  *
- * Purpose:	Finds the value that corresponds the the specified symbol
+ * Purpose:	Finds the value that corresponds the specified symbol
```
Comment (lrknox) [inline src/H5Tenum.c]:
```suggestion
 * Purpose:	Finds the value that corresponds to the specified symbol
```

File: src/H5Tpublic.h [hunk 8]
```diff
@@ -1138,7 +1138,7 @@ H5_DLL herr_t H5Tlock(hid_t type_id);
  * \ingroup H5T
  *
  * \brief Commits a transient datatype, linking it into the file and creating
- *        a new committed datatype
+ *        a newly committed datatype
```
Comment (lrknox) [inline src/H5Tpublic.h]: Judging from only this sentence, it seems that a committed datatype that did not yet exist will be created.  If true, I think new is better.   I also think that after it's created, newly-committed makes sense in **both** lines 1161 and 1164.  Seeing newly in line 1238, though, the whole process being described should be considered.

**Review by lrknox — CHANGES_REQUESTED** [review]

File: src/H5Tpublic.h (hunk 8 above)
Comment (brtnfld) [reply src/H5Tpublic.h]: fixed

File: src/H5Spublic.h (hunk 5 above)
Comment (brtnfld) [reply src/H5Spublic.h]: fixed.

**Review by lrknox — APPROVED** [review]

**Pull request merged**